digitalidentity/service.go

```diff
@@ -113,3 +114,34 @@ func CreateShareQrCode(httpClient requests.HttpClient, sessionID string, clientS
 	err = json.Unmarshal(responseBytes, qrCode)
 	return qrCode, err
 }
+
+// Get Share QR info using the supplied sessionID
+func GetShareQr(httpClient requests.HttpClient, sessionID string, clientSdkId, apiUrl string, key *rsa.PrivateKey) (share ShareSessionFetchedQrCode, err error) {
```

Comment:
```suggestion
// GetShareSessionQrCode is used to fetch the qr code by id.
func GetShareSessionQrCode(httpClient requests.HttpClient, qrCodeId string, clientSdkId, apiUrl string, key *rsa.PrivateKey) (share ShareSessionQrCode, err error) {
```

1. func name
2. comment
3. function arg should be `qrCodeId` not `sessionID`

Reply:
updated